bot/handlers/sending_videos/compile_clips_handler.py

```diff
@@ -60,79 +65,156 @@ async def _do_handle(self, message: Message) -> None:
             return await self.__reply_no_previous_search_results(message)
 
         segments = json.loads(last_search.segments)
+        selected_segments, errors = self.__parse_segments(content[1:], segments)
 
-        selected_segments = self.__parse_segments(content[1:], segments)
+        if errors:
+            return await self._answer(message, "\n".join(errors))
 
         if not selected_segments:
             return await self.__reply_no_matching_segments_found(message)
 
-        if not await DatabaseManager.is_admin_or_moderator(message.from_user.id) and len(selected_segments) > settings.MAX_CLIPS_PER_COMPILATION:
-            await self._answer(message,get_max_clips_exceeded_message())
-            return
+        if (
+            not await DatabaseManager.is_admin_or_moderator(message.from_user.id)
+            and len(selected_segments) > settings.MAX_CLIPS_PER_COMPILATION
+        ):
+            return await self._answer(message, get_max_clips_exceeded_message())
 
         total_duration = 0
         for segment in selected_segments:
-            duration = (segment["end"] + settings.EXTEND_AFTER) - (segment["start"] - settings.EXTEND_BEFORE)
+            duration = (
+                (segment["end"] + settings.EXTEND_AFTER)
+                - (segment["start"] - settings.EXTEND_BEFORE)
+            )
             total_duration += duration
             await self._log_system_message(
-                logging.INFO, f"Selected clip: {segment['video_path']} "
+                logging.INFO,
+                f"Selected clip: {segment['video_path']} "
                 f"from {segment['start']} to {segment['end']} with duration {duration}",
             )
-            await self._log_system_message(logging.INFO, f"Total duration: {total_duration}")
 
-        if await self._handle_clip_duration_limit_exceeded(message, total_duration):
+        if await self._check_clip_duration_limit(message, total_duration):
             return
```

Comment:
jakaś odpowiedź chyba musi być bo trochę to krzywe

Reply:
![image](https://github.com/user-attachments/assets/f01c45cc-a559-422a-9607-76f5eedf930e)
znaczy je

Reply:
Nie podoba mi się schowanie tego reply do tej funkcji

check duration limit powinno przyjmować tylko userid, a jeżęli zwraca true to zamiast return zrobić return await reply_something

bo teraz jest ukryte działanie i trochę dziwnie to wygląda